recipes/default.rb

```diff
@@ -18,10 +18,14 @@
 # limitations under the License.
 #
 
-build_essential 'htpasswd' do
+declare_resource(:build_essential, 'htpasswd') do
   compile_time true
 end
```

Comment:
The problem on Debian 11 is the /bin doesn't point to /usr/bin so it can't find `/bin/install`. A hackish way around this is to just create a symlink if `/bin/install` doesn't exist at compile time.
```suggestion
end

declare_resource(:link, '/bin/install') do
  to '/usr/bin/install'
  compile_time true
  not_if { ::File.exist? '/bin/install' }
end
```